lib/srv/sess.go

```diff
@@ -185,9 +218,50 @@ func (s *SessionRegistry) Close() {
 	s.log.Debug("Closing Session Registry.")
 }
 
+type sudoersCloser struct {
+	username     string
+	userSessions *userSessions
+	cleanup      func(name string) error
+}
+
+func (sc *sudoersCloser) Close() error {
+	count := sc.userSessions.del(sc.username)
```

Comment:
Do we clean up sudoers entry only upon session exit now? What if Teleport process crashes?

We should clean up before session start as well - was this removed or did I miss it?